tasks/update-granules-cmr-metadata-file-links/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@cumulus/update-granules-cmr-metadata-file-links",
+  "version": "2.0.1",
+  "description": "Update cmr metadata files with correct online access urls and etags and transfer etag info to granules' CMR files",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "homepage": "https://github.com/nasa/cumulus/tree/master/tasks/update-granules-cmr-metadata-file-links",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nasa/cumulus"
+  },
+  "publishConfig": {
+    "access": "public"
```

Comment:
I'm not sure we want to publish new tasks, given they're being pulled into TF via our release artifacts.   @nemreid  @markdboyd ?

Reply:
I think I would defer to publishing until we make a judgment call / change about all the tasks as a whole

Reply:
I'm 'ok' with that, but let's get it on the architecture meeting for a discussion.